src/auth/auth.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { AuthCredentialsDto } from './dto/auth-credential.sto';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @Post('/signup')
```

Comment:
Endpoint should be `/auth/register`

```suggestion
  @Post('/register')
```